README.md

```diff
@@ -15,83 +16,212 @@ npm run test
+		TestResults.addResult({component: component, type: 'FID', actualValue: Math.round((actualFirstInput + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'CLS', actualValue: Math.round((actualCLS + Number.EPSILON) * 1000) / 1000});
 
-		// Update
-		const actualUpdateTime = Update(filename, 'Changeable');
-		TestResults.addResult({component: 'Picker', type: 'Update', actualValue: actualUpdateTime});
+		expect(actualFirstInput).toBeLessThan(maxFID);
+		expect(actualCLS).toBeLessThan(maxCLS);
+	});
 
+	it('should have a good DCL, FCP and LCP', async () => {
+		const filename = getFileName(component);
+
+		let passContDCL = 0;
+		let passContFCP = 0;
+		let passContLCP = 0;
+		let avgDCL = 0;
+		let avgFCP = 0;
+		let avgLCP = 0;
+		for (let step = 0; step < stepNumber; step++) {
+			const dropdownPage = await testMultiple.newPage();
+
+			await dropdownPage.tracing.start({path: filename, screenshots: false});
+			await dropdownPage.goto('http://localhost:8080/dropdown');
+			await dropdownPage.waitForSelector('#dropdown');
+			await dropdownPage.waitForTimeout(200);
+
+			await dropdownPage.tracing.stop();
+
+			const {actualDCL, actualFCP, actualLCP} = PageLoadingMetrics(filename);
+			avgDCL = avgDCL + actualDCL;
+			if (actualDCL < maxDCL) {
+				passContDCL += 1;
+			}
+
+			avgFCP = avgFCP + actualFCP;
+			if (actualFCP < maxFCP) {
+				passContFCP += 1;
+			}
+
+			avgLCP = avgLCP + actualLCP;
+			if (actualLCP < maxLCP) {
+				passContLCP += 1;
+			}
+
+			await dropdownPage.close();
+		}
+		avgDCL = avgDCL / stepNumber;
+		avgFCP = avgFCP / stepNumber;
+		avgLCP = avgLCP / stepNumber;
+
+		TestResults.addResult({component: component, type: 'DCL', actualValue: Math.round((avgDCL + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'FCP', actualValue: Math.round((avgFCP + Number.EPSILON) * 1000) / 1000});
+		TestResults.addResult({component: component, type: 'LCP', actualValue: Math.round((avgLCP + Number.EPSILON) * 1000) / 1000});
+
+		expect(passContDCL).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgDCL).toBeLessThan(maxDCL);
+
+		expect(passContFCP).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgFCP).toBeLessThan(maxFCP);
+
+		expect(passContLCP).toBeGreaterThan(passRatio * stepNumber);
+		expect(avgLCP).toBeLessThan(maxLCP);
 	});
+});
 ```
 
-### Mount
+### Performance Chart App
 
-Mounting is very is pretty simple. We just need to find the top component for the component we're trying to test like `Mount(filename, 'Changeable');`. We can check the React Devtools to see which component is at the top of a specific component.
+We created an app to display metrics in charts that is available in `performanceMetrics` folder and can be run with `npm run serve`. 
+This way we can view the performance of each individual components at a given date on released version or development branch.

```

Comment:
I would add a

NOTE: This app is meant to be run on a dedicated server where all the performance tests results are saved. In order to have meaningful data on the local machine to test this app, please add the tests results in the `public` folder as following:
1. a folder for each enact released version
2. a folder named `develop`
3. in the `develop` folder, add a folder for every test run
![image](https://user-images.githubusercontent.com/63335068/178251764-303f37a8-4bb3-4509-8edf-739c3490d191.png)

4. Each of the created folders (for a version or a date in the develop branch) would contain txt files which contain the tests results data for each component
![image](https://user-images.githubusercontent.com/63335068/178251843-6e0007b0-535f-45d4-81e5-054f7db83703.png)

5. A file named developTestDate.txt . It contains the names of each folders located in the `develop` folder
![image](https://user-images.githubusercontent.com/63335068/178252140-9dad6efb-63f5-404f-9652-eda9a147e47d.png)

6. A file named releaseVersions.txt . It contains all the released version folde names
![image](https://user-images.githubusercontent.com/63335068/178252394-f275ea5b-bb38-4e95-bd57-452b4014ab8a.png)

Reply:
@jeonghee27, this part is very technical. Do we want to have this in the documentation? Please give your opinion too

Reply:
I thought there should be a guide on how to use performanceMetrics. It is a part your scrum can choose whether to do it together in this PR or do separately.

How about adding it as a new README.md file under the performanceMetrics/  path?
I don't think the above description you wrote is very technical.
It would be nice to add this.